Sender/Test_Sender.cpp

```diff
@@ -0,0 +1,17 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "test/catch.hpp"
+#include "BMS_AttributeDataSender.h"
+
+TEST_CASE("To check if file was successfully opened to read") 
```

Comment:
No test to see if the file could not be opened? How do you know if that code works?